<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>634</width>
     <height>374</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>6666666</width>
     <height>6666666</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="scanButton">
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QPushButton" name="undoButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Undo selecting</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="deleteButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete selected</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="selectButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Select all</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="dumpButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Dump</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="collapse">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Collapse</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="expand">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Expand</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="buttonSearch">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Search File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSelect">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Select File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFile">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="maximumSize">
             <size>
              <width>263</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="continueButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTreeWidget" name="treeWidget">
              <attribute name="headerMinimumSectionSize">
               <number>100</number>
              </attribute>
              <attribute name="headerStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Filename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Size</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="switchButton">
          <property name="text">
           <string>Go to Log and Errors</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionScan_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScan_Directory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionScan_Directory">
   <property name="text">
    <string>&amp;Scan Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
